lambdas/drivingpermitcheck/src/test/java/uk/gov/di/ipv/cri/drivingpermit/api/service/FormDataValidatorTest.java

```diff
@@ -42,7 +42,7 @@ void testFormDataValidatorNamesCannotBeNull() {
 
         LOGGER.info(validationResult.getError().toString());
 
-        assertEquals(TEST_STRING, drivingPermitForm.getForenames());
+        assertEquals(null, drivingPermitForm.getForenames());
```

Comment:
Why make this null but still use TEST_STRING below?

Reply:
Now I look this is an odd comparison why not just do
assertNull(drivingPermitForm.getForenames())